torch/_dynamo/utils.py

```diff
@@ -928,6 +1081,15 @@ class CompilationMetrics:
 
 
 def add_compilation_metrics_to_chromium(c: CompilationMetrics) -> None:
+    """
```

Comment:
Are you planning to remove this function entirely? If, so maybe a TODO and a comment to that affect would bolster your advice here?